src/web/metrics.rs

```diff
@@ -72,29 +78,64 @@ lazy_static::lazy_static! {
     )
     .unwrap();
 
-    pub static ref CONCURRENT_DB_CONNECTIONS: IntGauge = register_int_gauge!(
-        "docsrs_db_connections",
-        "The number of currently used database connections"
+    pub static ref USED_DB_CONNECTIONS: IntGauge = register_int_gauge!(
+        "docsrs_used_db_connections",
+        "The number of used database connections"
+    )
+    .unwrap();
+
+    pub static ref IDLE_DB_CONNECTIONS: IntGauge = register_int_gauge!(
+        "docsrs_idle_db_connections",
+        "The number of idle database connections"
+    )
+    .unwrap();
+
+    pub static ref OPEN_FILE_DESCRIPTORS: IntGauge = register_int_gauge!(
+        "docsrs_open_file_descriptors",
+        "The number of currently opened file descriptors"
+    )
+    .unwrap();
```

Comment:
What's the reason for monitoring this in the application directly instead of through the machine it's running on? It seems a little unreliable to try and measure it ourselves, and going through procfs can just as easily be done on the host.

Reply:
I looked into having `node_exporter` do it, but it only gave system wide statistics. See discord for more discussion